src/dbnode/server/server.go

```diff
@@ -90,6 +93,7 @@ const (
 	bgProcessLimitInterval           = 10 * time.Second
 	maxBgProcessLimitMonitorDuration = 5 * time.Minute
 	filePathPrefixLockFile           = ".lock"
+	serviceName                      = "m3dbnode"
```

Comment:
hm should this be config driven?

Reply:
Yeah we can and just default to `m3dbnode` if none is provided.